core/gql/gql_mutations/mutation_by_filter.py

```diff
@@ -150,3 +185,36 @@ def wrapper(cls, user, **data):
             async_mutate(cls, user, **data)
         return wrapper
     return inner_function
+
+
+def _append_filter_amount(amount_from, amount_to):
+    # TODO make a signal to contract module to not break modular approach and
+    #  not repeat the same code from contract module
+
+    status_notified = [1, 2]
```

Comment:
These numbers are difficult to understand. What is their meaning? Should you create meaningful constants and use them here?